lib/paperclip.rb

```diff
@@ -55,6 +55,7 @@
 require 'paperclip/has_attached_file'
 require 'paperclip/attachment_registry'
 require 'paperclip/filename_cleaner'
+require 'paperclip/rails_environment'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.